vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ version directory, and then changes are introduced.
 
 ## [v3.2.1] WIP
 
+### Changed
+- Enable aggregation layer in Kubernetes API server
```

Comment:
Just FYI. IMO it would be nice in the future to track the PR links here.

Reply:
can you give me an example?

Reply:
```
- Enable aggregation layer in Kubernetes API server (https://github.com/giantswarm/k8scloudconfig/pull/322)
```